src/html/types.rs

```diff
@@ -61,10 +61,13 @@ pub(crate) fn render_type_def(
       let name = if let Some(href) = href {
         format!(
           r#"<a href="{href}" class="link">{}</a>"#,
```

Comment:
`href` is safe because we generate it?

Reply:
yes, though we can escpae it for good measure